tools/lkl/Makefile.autoconf

```diff
@@ -135,6 +135,7 @@ endef
 define kasan_enable
   $(call set_autoconf_var,KASAN,y)
   $(call set_kernel_config,KASAN,y)
+  $(call set_kernel_config,KASAN_GENERIC,y)
```

Comment:
I've double checked and I don't think this is required, even for LLVM it is set by default on current master branch:

```
$ make -C tools/lkl/ LLVM=1 CROSS_COMPILE=x86_64-linux-gnu kasan=yes kasan_test=yes -j96
$ grep -i kasan .config
CONFIG_KASAN_SHADOW_OFFSET=0x0000200000000000
CONFIG_KASAN_SHADOW_SIZE=0x0000100000000000
CONFIG_HAVE_ARCH_KASAN=y
CONFIG_CC_HAS_KASAN_GENERIC=y
CONFIG_CC_HAS_KASAN_SW_TAGS=y
CONFIG_KASAN=y
CONFIG_KASAN_GENERIC=y
CONFIG_KASAN_OUTLINE=y
# CONFIG_KASAN_INLINE is not set
# CONFIG_KASAN_STACK is not set
CONFIG_KASAN_KUNIT_TEST=y

```

Reply:
Yes, I confirm the same behavior on my side too and, thus, extra config line in Makefile.autoconfig is redundant -- removing it. It seems the problem I didn't see it earlier was related to not cleaned .config file (I've opened an issue https://github.com/lkl/linux/issues/517 to track proper cleaning of .config files).